bridges/snowbridge/primitives/inbound-queue/src/v2/message.rs

```diff
@@ -64,6 +71,23 @@ impl core::fmt::Debug for IGatewayV2::EthereumAsset {
 	}
 }
 
+impl core::fmt::Debug for IGatewayV2::Xcm {
+	fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
+		f.debug_struct("Xcm")
+			.field("kind", &self.kind)
+			.field("data", &self.data)
+			.finish()
+	}
+}
+
+#[derive(Clone, Encode, Decode, RuntimeDebug, TypeInfo)]
+pub enum XcmCommand {
+	/// Represents raw XCM bytes
+	Raw(Vec<u8>),
+	/// A token registration template
+	TokenRegistration { token: H160, network: u8 },
```

Comment:
Prefer the name `CreateAsset` to match xcm/substrate terminology.

Oh and I forgot that we need to include ERC20 token metadata.

```suggestion
	CreateAsset { token: H160, network: u8, name: Vec<u8>, symbol: Vec<u8>, decimals: u8 },
```

Reply:
Added fields and set metadata call in https://github.com/vgeddes/polkadot-sdk/pull/13/commits/194ab7c9a4b932525b28975610a8a53d3100f06c. It would probably be better to do the create asset and set metadata call in one transact, will see if I can do it.

Reply:
The two Transact calls use different origin_kinds, so I think it needs to be 2 separate Transact calls.